content/version/2/0/code_of_conduct.md

```diff
@@ -11,15 +11,19 @@ We as members, contributors, and leaders pledge to make participation in our com
 
 We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
 
+We believe that a diverse community where people treat each other with respect is stronger, more vibrant, and has more potential contributors and more sources for ideas. We aim for more diversity.
```

Comment:
"diverse, welcoming, and inclusive"? Also, please drop "we aim for…" as that seems redundant?

Reply:
Done.